openlibrary/plugins/openlibrary/pages/config_edition.page

```diff
@@ -177,90 +541,166 @@
+            "name": "zdb-id",
+            "label": "ZDB-ID"
+        },
+        {
+            "url": "https://kansalliskirjasto.finna.fi/Record/vaari.@@@",
+            "website": "https://www.kansalliskirjasto.fi/",
+            "notes": "The National Library of Finland",
+            "name": "fennica",
+            "label": "Fennica"
+        },
+        {
+            "url": "https://opacplus.bsb-muenchen.de/metaopac/search?id=@@@",
+            "website": "http://www.bsb-muenchen.de",
             "notes": "",
-            "name": "issn",
-            "label": "ISSN"
+            "name": "bayerische_staatsbibliothek",
+            "label": "Bayerische Staatsbibliothek BSB-ID"
         },
         {
-            "website": "http://www.alibris.com",
+            "url": "https://www.abebooks.de/servlet/BookDetailsPL?bi=@@@",
+            "website": "https://www.abebooks.de",
             "notes": "",
-            "name": "alibris",
-            "label": "Alibris"
+            "name": "abebooks,de",
+            "label": "Abebooks.de"
         },
         {
-            "website": "http://www.isfdb.org/wiki/index.php/Publisher:Triad",
+            "url": "http://catalogo.bne.es/uhtbin/cgisirsi/x/0/0/57/5/3?searchdata1=@@@{CKEY}&user_id=WEBSERVER",
+            "website": "http://www.bne.es/en/Inicio/index.html",
             "notes": "",
-            "name": "isfdb",
-            "label": "isfdb"
+            "name": "dep\u00f3sito_legal",
+            "label": "Dep\u00f3sito Legal. Biblioteca Nacional de España"
         },
         {
-            "website": "http://www.elviejolibro.tk",
+            "website": "http://www.dcbooks.com/home",
             "notes": "",
-            "name": "libreria_virtual_el_viejo_libro",
-            "label": "Libreria Virtual El Viejo Libro"
+            "name": "dc_books",
+            "label": "DC Books"
         },
         {
-            "website": "",
+            "website": "http://www.publishamerica.com/",
+            "notes": "",
+            "name": "publishamerica",
+            "label": "PublishAmerica"
+        },
+        {
+            "url": "http://search.bl.uk/primo_library/libweb/action/display.do?doc=BLL01@@@",
+            "website": "http://www.bl.uk/bibliographic/natbib.html",
+            "notes": "",
+            "name": "british_national_bibliography",
+            "label": "British National Bibliography"
+        },
+        {
+            "url": "http://catalogue.bnf.fr/rechercher.do?motRecherche=@@@",
+            "website": "http://www.bnf.fr",
+            "notes": "",
+            "name": "biblioth\u00e8que_nationale_de_france_(bnf)",
+            "label": "Biblioth\u00e8que nationale de France (BnF)"
+        },
+        {
+            "url": "https://tools.wmflabs.org/reasonator/?q=@@@",
+            "website": "https://wikidata.org",
```

Comment:
:+1: I'm now thinking that we should create a similar authors.json with the author identifier config in the same kind of format, if we are happy with this method of storing data.

Reply:
As an admin/bot, can't we POST to `https://openlibrary.org/config/edition.json` just like any other record?

Reply:
Of course! Yes, it's not as difficult as I first thought, UI not required for now.

That's perhaps a new feature request for ol-client? Some sort of helper to enable config object uploads neatly, if we are planning to use this config mechanism more.

Currently you'll just have to load the json and use the session object:
`ol.session.post(<url>, <data>)`
That's not that hard really, we may want to add in some validation for some settings?